### ISSUE TITLE
Client Authentication Issue when using Extension in Chrome

### ISSUE THREAD

**Issue by adamlaws** [Microsoft/featuretimeline#1]
**Describe the bug**
When clicking on the Feature Timeline Tab, I receive a pop-up to enter my AD Credentials, after which time I get an error:

"TF400813: Resource not available for anonymous access. Client authentication required."

This appears to be a new problem, the previous version I was running last week didn't have this behaviour.

**Browser**
Choose a browser
[ ] IE11
[ X] Chrome
[ ] FireFox
[ ] Safari

**Version**
[ ] VSTS
[ X] TFS 2017 Update 2/3
[ ] TFS 2018

**Screenshots**

**Comment by navneet-g** [comment]
Can you upload fiddler trace? This could be related to network connectivity.

**Comment by adamlaws** [comment]
[FeatureTimeLineFiddler.zip](https://github.com/Microsoft/featuretimeline/files/1999994/FeatureTimeLineFiddler.zip)

Fiddler Trace attached - Created using Telerik Fiddler against Chrome

**Comment by adamlaws** [comment]
[FeatureTimeLineFiddlerEdge.zip](https://github.com/Microsoft/featuretimeline/files/2000344/FeatureTimeLineFiddlerEdge.zip)

Attached is also the trace from Edge.

This is the only Extension where we face this issue.

**Comment by adamlaws** [comment]
After running the URL, I receive what looks like a long string of JSON containing field names and WIT Definitions.

Yes, I'm running https on this server.

Not sure if it helps, but we also have a custom backlog level, so we have Epic, Requirement, Feature and Story in that order.

**Comment by adamlaws** [comment]
There may be two issues here, I quickly switched back to http and I got a different error:

An undefined error occurred while attempting to connect to the server. Status code 0: error.

Looks like someone else also found this problem as another issue, but mine appears linked to the https connection.

**Comment by navneet-g** [comment]
Thanks @adamlaws this seems related to network configuration it seems, as I am just using standard REST API clients to make a request .... in the other issue I mentioned about trying the solution at following link https://github.com/onlyutkarsh/ExportImportBuildDefinition/issues/9.

The custom backlog levels is not the issue as the code is not reaching main app logic at all and failing while loading data.

**Comment by adamlaws** [comment]
@navneet-g ,we're not using any Alias's in our TFS Configuration in the Dev Environment, just a direct FQDN and I also just tried the same steps using the IP Address directly, but with the same results for AN http connection

In our case we're running TFS 2017 Update 2.

It's strange because it was working fine when I first installed the extension, only after one of the updates did I get this problem.

The problem is limited to only this extension, other DevLabs ones are working fine. What would be the next steps to debug this?

**Comment by navneet-g** [comment]
I am not a network expert, I will ask around to see if I can find some help, 
By any chance do you remember the day it was working before? I can look up the change history to see if I can isolate the issue.

**Comment by adamlaws** [comment]
OK – So this is a strange situation.

I looked in the _oi Logs and could see the 401 error being returned (Even though Windows Authentication is disabled in IIS), so I enabled Windows Auth in IIS and it displayed fine, I then disabled it, and it still continues to work fine, even with the standard IIS Configuration.

So, seems that restarting IIS resolved the problem in the end.

**Comment by navneet-g** [comment]
Nice ... the classic restart solution :).
